photon-lib/src/main/scala/com/linkedin/photon/ml/function/L2Regularization.scala

```diff
@@ -174,7 +186,7 @@ trait L2RegularizationTwiceDiff extends TwiceDiffFunction with L2RegularizationD
       coefficients: Coefficients,
       multiplyVector: Coefficients,
       normalizationContext: BroadcastWrapper[NormalizationContext]): Vector[Double] =
-    super.hessianVector(input, coefficients, multiplyVector, normalizationContext) +
+    super.hessianVector(input, coefficients, multiplyVector, normalizationContext) +:+
```

Comment:
Why do we change to `+:+`? `+` supports element-wise addition

Reply:
Trying to remember - I think my train of thought was that `+` would be for scalar values, and `+:+` would be for vector values. I think I was trying to make it clear what type of values are being summed by looking at which symbol is being used.